bpfd-api/src/util.rs

```diff
@@ -32,9 +32,11 @@ pub mod directories {
     pub const RTDIR_FS_XDP: &str = "/run/bpfd/fs/xdp";
     pub const RTDIR_FS_MAPS: &str = "/run/bpfd/fs/maps";
     pub const RTDIR_PROGRAMS: &str = "/run/bpfd/programs";
+    pub const STDIR_SOCKET: &str = "/run/bpfd/sock";
+    pub const STPATH_BPFD_SOCKET: &str = "/run/bpfd/sock/bpfd.sock";
```

Comment:
nit: You can just leave them in `/run/bpfd`
No need for a subdirectory.

Reply:
Lol for sure!

Reply:
So I ended still having to create a `/run/bpfd/csi/` dir because I still needed a dedicated directory for the csi sock so that we could volume mount it separately in K8s.  
i.e today we do 

```
        - name: runtime
          mountPath: /run/bpfd
          mountPropagation: Bidirectional
``` 

and needed 

```
        - mountPath: /run/bpfd
           subPath: csi.sock
          name: socket-dir
```

as well which lead to daemonset issues.

However once we can configure this socket location (if we end up doing that) this issue will go away and by default it can all just go in /run/bpfd 👍  I hope that's alright